src/main/java/com/google/sps/servlets/NewMemberServlet.java

```diff
@@ -66,6 +69,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
         datastore.put(familyEntity);
 
+        String calendarID = (String) familyEntity.getProperty("calendarID");
+
+        if(calendarID != null) {
```

Comment:
nit: Add a comment before this block to indicate that this adds the user to the calendar.